### ISSUE TITLE
关于Synapse数据集划分问题

### ISSUE THREAD

**Issue by Xavier4j** [282857341/nnFormer#91]
你好，非常感谢你们的工作，关于Synapse数据集划分我有一个问题，我看到你说
`It seems that we use the whole data to train, but we will split the validation set from the training set`
但是，你在训练的时候会通过验证集的结果来保存best模型不是么，那这样的话，不就等于在test数据集上进行训练了吗
希望得到你的回答，感谢！

**Comment by 282857341** [comment]
我们这是follow前人的工作，前人在这个这个数据集上也是这么做的

**Comment by Xavier4j** [comment]
非常感谢您的回复，那请问，这个前人是哪篇论文呀，谢谢

**Comment by xiaoc57** [comment]
> 

这个应该从transunet那篇文章开始就这么做了
